src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritDynamicUrlProcessor.java

```diff
@@ -246,13 +245,6 @@ private static String regexEscapted(char symbol) {
     public static List<GerritProject> fetch(String gerritTriggerConfigUrl, String serverName)
             throws IOException, ParseException {
 
-        if (gerritTriggerConfigUrl == null) {
```

Comment:
I see no reason to remove this, it will just cause another `MalformedURLException` to be thrown from the URL constructor with a more cryptic message.

Reply:
Because updateTriggerConfigURL was trigged only from once place and now we have a check directly in this place.
Should I keep it to double check?

Reply:
Well it's a public method so it's best to keep it I think :)